packages/dds/cell/src/cell.ts

```diff
@@ -158,6 +205,13 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>> impl
         return this.data === undefined;
     }
 
+    /**
+     * {@inheritDoc ISharedCell.getAttributor}
+     */
+    public getAttributor(): CellAttributor {
```

Comment:
i think the API on SharedCell should probably just expose a way to get an `AttributionKey`.